sdk/storage/azure-storage-file-datalake/CHANGELOG.md

```diff
@@ -3,6 +3,8 @@
 ## 12.17.0-beta.1 (Unreleased)
 
 ### Features Added
+- Added `DataLakeDirectoryClient.deleteRecursively()` which allows users to delete a directory and all of its contents recursively. 
+- Added support for paginated directory delete when using AAD authentication. Note that this feature only applies to HNS storage accounts and when using token authentication.
```

Comment:
It might be worth adding a link to documentation that explains what HNS is.

Reply:
Will add it when merging to main.